docs/updating/migration-to-cdn/react.md

```diff
@@ -0,0 +1,461 @@
+---
+menu-title: React
+meta-title: React CKEditor 5 - migrate integration from npm to CDN | CKEditor 5 documentation
+meta-description: Migrate React CKEditor 5 integration from npm to CDN in a few simple steps. Learn how to install React CKEditor 5 integration in your project using the CDN.
+category: migrations
+order: 30
+---
+
+# Migrating CKEditor&nbsp;5 React integration from npm to CDN
+
+This guide will help you migrate CKEditor&nbsp;5 React integration from an NPM-based installation to a CDN-based installation.
+
+## Prerequisites
+
+Remove the existing CKEditor&nbsp;5 packages from your project. If you are using the NPM-based installation, you can remove it by running the following command:
+
+```bash
+npm uninstall ckeditor5 ckeditor5-premium-features
+```
+
+Upgrade the CKEditor&nbsp;5 React integration to the latest version. You can find the latest version in the {@link getting-started/integrations-cdn/react-default-cdn React} documentation.
+
+Ensure that your testing suite uses real web browser environments for testing. If you are using `jsdom` or any other environment without a real DOM, you may need to adjust the testing suite configuration to use a real browser because CDN scripts injection might be not recognized properly in such environments.
+
+## Migration steps
+
+### Migrate `CKEditor` component
+
+#### Step 1: Remove CKEditor&nbsp;5 imports
+
+If you have any CKEditor&nbsp;5 imports in your test files, remove them. For example, remove lines like:
+
+```javascript
+import { ClassicEditor, ... } from 'ckeditor5';
+import { AIAdapter, ... } from 'ckeditor5-premium-features';
+```
+
+#### Step 2: Use `useCKEditorCloud` hook to load CKEditor&nbsp;5 from CDN
+
+`useCKEditorCloud` is a hook that allows you to load CKEditor&nbsp;5 from the CDN. It returns an object with the `CKEditor` and `CKEditorPremiumFeatures` properties. Here is an example of migrating the basic CKEditor&nbsp;5 React component:
```

Comment:
Maybe it's just me, but starting a sentence with an inline code looks a bit strange. So, how about?
```suggestion
The `useCKEditorCloud` function is a hook that allows you to load CKEditor&nbsp;5 from the CDN. It returns an object with the `CKEditor` and `CKEditorPremiumFeatures` properties. Here is an example of migrating the basic CKEditor&nbsp;5 React component:
```